pkgs/development/python-modules/adb-homeassistant/default.nix

```
@@ -0,0 +1,39 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, libusb1
+, rsa
+, pycryptodome
+, pytest
+, mock
+}:
+buildPythonPackage rec {
+  pname = "adb-homeassistant";
+  version = "1.3.1";
+
+  src = fetchFromGitHub {
+    owner = "JeffLIrion";
+    repo = "python-adb";
+    rev = "5949bf432307cbba7128e84d7bc6add7f054a078";
+    sha256 = "0s3fazvbzchn1fsvjrd1jl8w9y4dvvgq6q8m8p5lr2gri0npr581";
+
+  };
+
+  propagatedBuildInputs = [
+    libusb1
+    rsa
+    pycryptodome
+  ];
+
+  checkInputs = [ pytest mock ];
+  checkPhase = ''
+    py.test adb test
```

```suggestion
    py.test test
```
adb is just the dir whose coverage they want to compute